tests/virt/utils.py

```diff
@@ -355,3 +363,85 @@ def validate_machine_type(vm, expected_machine_type):
     assert vmi_xml_machine_type == expected_machine_type, (
         f"libvirt machine type {vmi_xml_machine_type} does not match expected type {expected_machine_type}"
     )
+
+
+def patch_hco_cr_with_mdev_permitted_hostdevices(hyperconverged_resource, supported_gpu_device):
+    with ResourceEditorValidateHCOReconcile(
+        patches={
+            hyperconverged_resource: {
+                "spec": {
+                    "mediatedDevicesConfiguration": {"mediatedDeviceTypes": [supported_gpu_device[MDEV_TYPE_STR]]},
+                    "permittedHostDevices": {
+                        "mediatedDevices": [
+                            {
+                                "mdevNameSelector": supported_gpu_device[MDEV_NAME_STR],
+                                "resourceName": supported_gpu_device[VGPU_DEVICE_NAME_STR],
+                            }
+                        ]
+                    },
+                }
+            }
+        },
```

Comment:
why it is different from original?
you don't set `nodeMediatedDeviceTypes` and second part of `mediatedDevices`